sockets/twitch.js

```diff
@@ -252,6 +268,31 @@ module.exports = function(io) {
             });
         }
 
+        function processFollowers() {
+            console.log('Processing new followers...');
+            /*var dummyFollower = {
+                user:{
+                    name:'Dummy Follower'
+                }
+            }
+
+            currentFollowers.unshift(dummyFollower);
+            console.log(currentFollowers);*/
+            var numNewFollowers = twitchData.twitchFollowers - numFollowersAtLaunch;
```

Comment:
So, because Twitch only sends us 25 followers at a time, I perform front-end validation to prevent displaying the same users anyway, but adding that fix will make this more efficient. I'll change it when I get back home this afternoon..